src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,203 @@
+
+    private static void printFormat(String... args) {
+        System.out.println(HORIZONTAL_LINE);
+        for (String arg : args) {
+            System.out.println(arg);
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    private static void intro() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        printFormat("Hello! I'm Duke",
+                "What can I do for you?",
+                "type 'commands' for the list of commands");
+    }
+
+    private static void handleInput(String input) 
+        throws SingleWordCommandException, IndexOutOfBoundsException, NoSlashException, AlreadyMarkedException,
+        AlreadyUnmarkedException, InvalidNumberException {
+        if (input.contains("list")) {
+            if (input.equals("list")) {
+                handleList();
+            } else {
+                throw new SingleWordCommandException();
+            }
+
+        } else if (input.contains("unmark")) {
+            handleUnmark(input);
+
+        } else if (input.contains("mark")) {
+            handleMark(input);
+
+        } else if (input.contains("todo")) {
+            handleToDo(input);
+
+        } else if (input.contains("deadline")) {
+            if (input.contains("/")) {
+                handleDeadline(input);
+            } else {
+                throw new NoSlashException();
+            }
+
+        } else if (input.contains("event")) {
+            if (input.contains("/")) {
+                handleEvent(input);
+            } else {
+                throw new NoSlashException();
+            }
+
+        } else if (input.contains("commands")) {
+            if (input.equals("commands")) {
+                handleCommand();
+            } else {
+                throw new SingleWordCommandException();
+            }
+
+        } else {
+            printFormat("OOPS!!! I'm sorry, but I don't know what that means :-(",
+                    "Please type in 'commands' if you are not sure of the commands");
+
+        }
+    }
+
+    private static void handleList() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < list.length; i++) {
+            if (list[i] != null) {
+                System.out.println(String.valueOf(i + 1) + "." + list[i]);
+            } else {
+                System.out.println(HORIZONTAL_LINE);
+                break;
+            }
+        }
+    }
+
+    private static void handleUnmark(String input) throws AlreadyUnmarkedException, InvalidNumberException {
+        int markInt = Integer.parseInt(input.substring(7)) - 1;
+        if (markInt + 1> taskCounter) {
```

Comment:
Try to standardize spaces before and after operators e.g markInt + 1 > taskCounter